x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_add.tsx

```diff
@@ -376,6 +381,19 @@ const RuleAdd = <
             }}
           />
         )}
+        {isShowRequestModalOpen && (
```

Comment:
Is there any way to only enable the button when the request is valid? For example, I can select an ES query rule type and show the request before selecting whether I want a KQL or DSL type. The modal shows the current request but if the user tries to use it, it won't work